tests/templates/kuttl/logging/helm-bitnami-postgresql-values.yaml.j2

```diff
@@ -6,11 +6,16 @@ volumePermissions:
 
 primary:
   podSecurityContext:
-{% if test_scenario['values']['openshift'] == 'true' -%}
```

Comment:
Was there any specific reason for this change? I have to admin I would prefer the current solution as it uses less lines and "indentation magic"

Reply:
The current/original solution did not work for openshift tests. I tried several different options and this was the only approach that I could get to work. If there is a better way I'm more than happy to change it.

Reply:
That's fine for me if the change is required :)

Reply:
I think I tied myself in knots here :) I've reverted it back to having no indentation-extras (just % instead of -% etc.) and it works in both environments.